homeassistant/components/ambient_station/sensor.py

```diff
@@ -523,30 +575,35 @@
         name="Wind Gust",
         icon="mdi:weather-windy",
         native_unit_of_measurement=SPEED_MILES_PER_HOUR,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_WINDSPDMPH_AVG10M,
         name="Wind Avg 10m",
         icon="mdi:weather-windy",
         native_unit_of_measurement=SPEED_MILES_PER_HOUR,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_WINDSPDMPH_AVG2M,
         name="Wind Avg 2m",
         icon="mdi:weather-windy",
         native_unit_of_measurement=SPEED_MILES_PER_HOUR,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_WINDSPEEDMPH,
         name="Wind Speed",
         icon="mdi:weather-windy",
         native_unit_of_measurement=SPEED_MILES_PER_HOUR,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_YEARLYRAININ,
         name="Yearly Rain",
         icon="mdi:water",
         native_unit_of_measurement=PRECIPITATION_INCHES,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
Is the total of rain that fell this whole year? Now that is a total_increasing

Reply:
Yes, it's for the whole year, but it will ostensibly reset then... Sorry, I'm finding when to apply `TOTAL_INCREASING` to be confusing.